packages/ic-response-verification/src/validation/common_validation.rs

```diff
@@ -1,141 +1,4 @@
-use candid::Principal;
```

Comment:
Moved to `packages/ic-certificate-verification/src/certificate_verification.rs`.